crates/wash-cli/tests/wash_secrets.rs

```diff
@@ -52,26 +52,102 @@ async fn test_secret_put_and_get() -> anyhow::Result<()> {
         .get("type")
         .is_some_and(|v| v == SECRET_TYPE));
     let serde_json::Value::String(policy) = retrieved_secret
-        .get("policy_properties")
-        .context("policy_properties should exist")?
+        .get("policy")
+        .context("policy should exist")?
     else {
-        bail!("policy_properties should be a string");
+        bail!("policy should be a string");
     };
 
-    let policy: HashMap<String, String> =
-        serde_json::from_str(policy).context("policy_properties should be a JSON object")?;
-    assert!(policy.get("type").is_some_and(|t| t == SECRET_POLICY_TYPE));
-    let Some(properties) = policy.get("properties") else {
-        bail!("policy_properties should have a 'properties' field");
+    let serde_json::Value::Object(policy) =
+        serde_json::from_str(policy).context("policy should be a JSON object")?
```

Comment:
```suggestion
    let policy =
        serde_json::from_str(policy).as_object().context("policy should be a JSON object")?
```

Reply:
This didn't quite work for me so I just left it as-is, considering it's just in the test